Samplero/Samplero/Global/Utility/LocalFileManager.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  LocalFileManager.swift
+//  Samplero
+//
+//  Created by JiwKang on 2022/10/13.
+//
+
+import Foundation
+import UIKit 
+
+class LocalFileManager {
+    
+    // MARK: - Properties
+
+    static let instance = LocalFileManager()
```

Comment:
p1;
싱글톤을 하는 게 목적이면 기본생성자를 private으로 해야 완전히 막을 수 있습니다! 😁
```suggestion
    static let instance = LocalFileManager()
    
    private init() { }
```

Reply:
아 그렇군요! 바로 적용시키겠습니다!
좋은 지적 감사합니다😮